tests/unit/test_data_cleaning_workflow.py

```
@@ -0,0 +1,272 @@
+    print(f"\nOriginal dtypes:\n{original_data.dtypes}")
+    
+    # Store original values for comparison
+    original_age_0 = original_data.loc[0, 'age']
+    original_weight_4 = original_data.loc[4, 'weight']
+    
+    print(f"\nStored original values:")
+    print(f"  age[0] = {original_age_0}")
+    print(f"  weight[4] = {original_weight_4}")
+    
+    # Apply cleaning
+    cleaned_df, report = clean_dataframe(original_data)
```

Ruff (0.14.13)

Unpacked variable `report` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)